mock-identity-system/src/main/java/io/mosip/esignet/mock/identitysystem/service/impl/AuthenticationServiceImpl.java

```diff
@@ -75,6 +82,9 @@ public class AuthenticationServiceImpl implements AuthenticationService {
     @Value("${mosip.esignet.authenticator.auth-factor.kba.field-language}")
     private String fieldLang;
 
+    @Value("#{${mosip.esignet.mock.authenticator.ida.oidc-claims}}")
```

Comment:
Lets rename the property as "mosip.mock.ida.identity-openid-claims-mapping"

Reply:
renamed